src/components/SearchModal/ManageLists.tsx

```diff
@@ -156,31 +160,39 @@ const ListRow = memo(function ListRow({ listUrl }: { listUrl: string }) {
           <StyledListUrlText active={isActive} mr="6px">
             {list.tokens.length} tokens
           </StyledListUrlText>
-          <StyledMenu ref={node as any}>
-            <ButtonEmpty onClick={toggle} ref={setReferenceElement} padding="0">
-              <Settings stroke={isActive ? theme.bg1 : theme.text} size={12} />
-            </ButtonEmpty>
-            {open && (
-              <PopoverContainer show={true} ref={setPopperElement as any} style={styles.popper} {...attributes.popper}>
-                <div>{list && listVersionLabel(list.version)}</div>
-                <SeparatorDark />
-                <ExternalLink href={`https://tokenlists.org/token-list?url=${listUrl}`}>
-                  <Trans>View list</Trans>
-                </ExternalLink>
-                <UnpaddedLinkStyledButton onClick={handleRemoveList} disabled={Object.keys(listsByUrl).length === 1}>
-                  <Trans>Remove list</Trans>
-                </UnpaddedLinkStyledButton>
-                {pending && (
-                  <UnpaddedLinkStyledButton onClick={handleAcceptListUpdate}>
-                    <Trans>Update list</Trans>
+          {isKyberList || (
+            <StyledMenu ref={node as any}>
```

Comment:
remove `any` here by updating:
`const node = useRef<HTMLDivElement>(null)`

Reply:
Leave it bro. Not my code and it still working fine.